### ISSUE TITLE
Provide isar-image-tiny

### ISSUE THREAD

**Issue by jan-kiszka** [ilbers/isar#24]
The docker image debian:stretch-slim is only 55 MB on x86 (granted, without kernel and modules), isar-image-base rootfs is 540 MB. It should be possible to provide a similarly slim image via Isar as well.

**Comment by ismagulb** [comment]
Indeed, the kernel takes 136 MB, df -h on qemuamd64-stretch reports 540 MB. So, 136 + 55 = 191 MB might in theory be achievable. We have to look what docker installs. What is the use case?

qemuamd64-stretch has:
104 packages
25 essential
54 required
21 important
5 standard
21 optional
3 extra

FWIW, an attempt to remove the extra libapparmor1 and liblz4-1 fails, since packages like dbus, systemd, etc. depend on those.

**Comment by jan-kiszka** [comment]
The use case is simply having a small working baseline for projects to derive from, rather than a larger base image to be shrunk when size matters (due to limited storage - or limited bandwidth when downloading update).

**Comment by ismagulb** [comment]
Hmm, to my knowledge, it already is -- we don't add extra packages. So, less is going to be stripping below the minimal installation calculated by multistrap. Let us check what docker installs and whether it's possible to install less with multistrap, or whether we'd have to remove packages in post-processing. If it's doable, it should become the base image.

**Comment by jan-kiszka** [comment]
Here is the list of stretch-slim:

```
adduser
apt
base-files
base-passwd
bash
bsdutils
coreutils
dash
debconf
debian-archive-keyring
debianutils
diffutils
dpkg
e2fslibs:amd64
e2fsprogs
findutils
gcc-6-base:amd64
gpgv
grep
gzip
hostname
init-system-helpers
libacl1:amd64
libapt-pkg5.0:amd64
libattr1:amd64
libaudit-common
libaudit1:amd64
libblkid1:amd64
libbz2-1.0:amd64
libc-bin
libc6:amd64
libcap-ng0:amd64
libcomerr2:amd64
libdb5.3:amd64
libdebconfclient0:amd64
libfdisk1:amd64
libgcc1:amd64
libgcrypt20:amd64
libgpg-error0:amd64
liblz4-1:amd64
liblzma5:amd64
libmount1:amd64
libncursesw5:amd64
libpam-modules:amd64
libpam-modules-bin
libpam-runtime
libpam0g:amd64
libpcre3:amd64
libselinux1:amd64
libsemanage-common
libsemanage1:amd64
libsepol1:amd64
libsmartcols1:amd64
libss2:amd64
libstdc++6:amd64
libsystemd0:amd64
libtinfo5:amd64
libudev1:amd64
libustr-1.0-1:amd64
libuuid1:amd64
login
lsb-base
mawk
mount
multiarch-support
ncurses-base
ncurses-bin
passwd
perl-base
sed
sensible-utils
sysvinit-utils
tar
tzdata
util-linux
zlib1g:amd64
```

**Comment by elfgoh** [comment]
Based on the image I built and running `df` command, the space used is 340M, which seems a little big for a base image. Please consider slimming it down :)

**Comment by stormc** [comment]
One target may be `/usr/share/locale` which accounts for 30,9 MiB.
Another worthwhile target may be `/var/cache/apt/archives` which accounts for ~118 MiB and only contains packages that are already installed, i.e., "waste".

**Comment by jan-kiszka** [comment]
Final "apt-get clean" missing?

**Comment by stormc** [comment]
It's the unused local**e** in `/usr/share/locale`, so a final `localepurge` should solve this as `/etc/locale.nopurge` is setup.

**Comment by stormc** [comment]
With commit `ec997ad`, waste induced by the apt cache and superfluous locales is fixed.

For "tiny" images, I'd like to see
- @jan-kiszka's `rm -rf /var/lib/apt/lists/*` accounting for 63MiB,
- some strip-down of `/lib/modules` which accounts for 181MiB,
- (optional?) removal of docs, e.g., `/usr/share/{man,doc}` which accounts for ~ 15MiB, and
- (optional?) cleanup of i18n and timezones `/usr/share/{i18n,zoneinfo}` accounting for ~17MiB
